packages/docusaurus/src/webpack/base.ts

```diff
@@ -137,7 +154,7 @@ export function createBaseConfig(
       splitChunks: isServer
         ? false
         : {
-            // Since the chunk name includes all origin chunk names it’s recommended for production builds with long term caching to NOT include [name] in the filenames
+            // Since the chunk name includes all origin chunk names it's recommended for production builds with long term caching to NOT include [name] in the filenames
             name: false,
             cacheGroups: {
```

Comment:
drive by: I believe the styles chunk here should switch to `test: module => module.type === 'css/mini-extract'`, see: https://github.com/webpack-contrib/mini-css-extract-plugin#extracting-all-css-in-a-single-file

Reply:
Thanks,

in practice, it seems both still work but let's use the new `type` as it seems usage is documented for Webpack 5